### PR TITLE
Refactor HTTP server connector actions

### PR DESCRIPTION
**Pull request by chamil321**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/Redirect.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        HTTPCarbonMessage inboundRequestMsg =
+                (HTTPCarbonMessage) bConnector.getNativeData(HttpConstants.TRANSPORT_MESSAGE);
+
+        BStruct outboundResponseStruct = (BStruct) context.getRefArgument(1);
+        HTTPCarbonMessage outboundResponseMsg = HttpUtil
+                .getCarbonMsg(outboundResponseStruct, HttpUtil.createHttpCarbonMessage(false));
+
+        BEnumerator code = (BEnumerator) context.getRefArgument(2);
+
+        if (code.stringValue().equals(RedirectCode.MULTIPLE_CHOICES_300.toString())) {
```
Comment (pamod) [inline stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/Redirect.java]: Can we replace this with switch statements ? (given RedirectCode is an enum)

Comment (chamil321) [reply stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/Redirect.java]: Fixed by b15e49c

File: stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/RespondContinue.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @Override
+    public void execute(Context context) {
+        BConnector bConnector = (BConnector) context.getRefArgument(0);
+        HTTPCarbonMessage inboundRequestMsg =
+                (HTTPCarbonMessage) bConnector.getNativeData(HttpConstants.TRANSPORT_MESSAGE);
+
+        BStruct outboundResponseStruct = BLangConnectorSPIUtil.createBStruct(context.getProgramFile(),
+                PROTOCOL_PACKAGE_HTTP, HttpConstants.RESPONSE);
+        HTTPCarbonMessage outboundResponseMsg = HttpUtil.createHttpCarbonMessage(false);
+        outboundResponseMsg.setProperty(HttpConstants.HTTP_STATUS_CODE, 100);
```
Comment (VijithaEkanayake) [inline stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/RespondContinue.java]: Better to use a constant here.

Comment (chamil321) [reply stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpserverconnector/RespondContinue.java]: Fixed by e4e7f07

**Comment by shafreenAnfar** [comment]
Closed this as this is not needed anymore.

**Pull request closed without merging**